api/v1/model_manager_service.proto

```diff
@@ -114,13 +114,16 @@ enum QuantizationType {
 }
 
 message RegisterModelRequest {
+  string id = 7;
```

Comment:
Could you add a comment to describe how this field works?

Reply:
This field allows a user to name the model arbitrarily. The name of the model downloaded from HF can be in any format, we don't have to converted its name into "base-model-name.suffix" format.  

This ID would be also the model name to be used in inference request.

Reply:
Ah yes, I understand. My comment was to add the comment in the code as the API contract is not very straightforwad.